config/initializers/0_constants.rb

```diff
@@ -1,6 +1,7 @@
 # File is prefixed with 0_ so that it loads first
 
 APP_NAME = "Example App"
-
+COMPANY_NAME = "Identity Square, Inc."
+COPYRIGHT_DATE = "2020"
```

Comment:
it would just be the current year. this does not need to be here. just print the current year in ruby in the place where this is getting rendered. this won't be in the constants file